### DISCUSSION TITLE
Official documentation of Trino python client

### DISCUSSION THREAD

**Discussion by Ouuumar** [trinodb/trino-python-client#350] [General]
Hello, I am looking for the documentation of this module, does it exist ? I can't find it

**Comment by hashhar** [comment]
The README at https://github.com/trinodb/trino-python-client contains all the documentation.

**Reply by Ouuumar** [reply]
All right thank you, I was looking for a more detailed like this for any `functions` https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.html.  But good enough thanks for the work :) !

**Reply by hashhar** [reply]
All the functions that the client provides are standard Python db-api functions so you can find them at https://peps.python.org/pep-0249/
